### PR TITLE
240 NPSP Localization

### PR DESCRIPTION
**Pull request by dmkud**
# Critical Changes

# Changes

- NPSP now supports Brazilian Portuguese.

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/translations/pt_BR.translation-meta.xml [hunk 1]
```diff
@@ -1164,7 +1164,7 @@
     </customLabels>
     <customLabels>
         <name>RD2_EntryFormInsertSuccessMessage</name>
-        <label>A Doação recorrente "{0}” foi criada.</label>
+        <label>A Doação recorrente &quot;{0}” foi criada.</label>
```
Comment (lparrott) [inline force-app/main/default/translations/pt_BR.translation-meta.xml:1167]: This is *probably* fine, but the second quote is a "rich text" version and could be replaced by an HTML entity.

File: force-app/main/default/translations/pt_BR.translation-meta.xml [hunk 2]
```diff
@@ -1237,7 +1237,7 @@
     </customLabels>
     <customLabels>
         <name>RD2_EntryFormUpdateSuccessMessage</name>
-        <label>A Doação recorrente "{0}” foi salva.</label>
+        <label>A Doação recorrente &quot;{0}” foi salva.</label>
```
Comment (lparrott) [inline force-app/main/default/translations/pt_BR.translation-meta.xml:1240]: Same as above.

**Review by lparrott — APPROVED** [review]
Found some rogue quote characters, nothing major!

File: force-app/main/default/translations/pt_BR.translation-meta.xml (hunk 1 above)
Comment (npsp-reedestockton) [reply force-app/main/default/translations/pt_BR.translation-meta.xml:1167]: This _is_ an entity in the English version.  It should be changed.

File: force-app/main/default/translations/pt_BR.translation-meta.xml (hunk 2 above)
Comment (npsp-reedestockton) [reply force-app/main/default/translations/pt_BR.translation-meta.xml:1240]: This should be changed to an entity as well.

File: force-app/main/default/translations/pt_BR.translation-meta.xml [hunk 3]
```diff
@@ -0,0 +1,8026 @@
...
+        <name>conMergeTooManyContactError</name>
+        <label>Muitos contatos duplicados. Use a opção Pesquisar contato para mesclar contatos</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSearchConText</name>
+        <label>Procure Contatos específicos para ver se eles têm duplicatas</label>
+    </customLabels>
+    <customLabels>
+        <name>conMergeSeeDupConText</name>
+        <label>Veja todos os Contatos duplicados encontrados usando regras de correspondência. &lt;a href=&apos;https://powerofus.force.com/IP_Contacts&apos; target=&apos;_blank&apos;&gt;&lt;b&gt; Conte-me mais &lt;/b&gt;&lt;/a&gt;</label>
```
Comment (npsp-reedestockton) [inline force-app/main/default/translations/pt_BR.translation-meta.xml:3369]: This was changed to an html quote entity in a recent bug PR.  Unless this is changed to match, the url will be stripped.

This is quite likely to be an issue in all translations.

**Review by npsp-reedestockton — CHANGES_REQUESTED** [review]

**Comment by daniel-fuller** [comment]
@npsp-reedestockton this one should be ready for another pass

**Review by lparrott — APPROVED** [review]

**Review by npsp-reedestockton — APPROVED** [review]
That's great @daniel-fuller!  Thanks!!

**Pull request closed without merging**